presto-spi/src/main/java/com/facebook/presto/spi/plan/CteConsumerNode.java

```diff
@@ -104,7 +104,7 @@ public String getCteId()
         return cteId;
     }
 
-    private static void checkArgument(boolean condition, String message)
+    public static void checkArgument(boolean condition, String message)
```

Comment:
This method doesn't really belong here and should not be public. There are better supported libraries to handle this sort of thing, e.g. Preconditions in Guava. and it's trivial to inline.

https://jlbp.dev/JLBP-1